README.md

```diff
@@ -87,7 +87,7 @@ yarn install rn-indy-sdk
 
 ### Using the framework
 
-While the framework is still in early development the best way to know what API the framework exposes is by looking at the [tests](src/__tests__), the [source code](src) or the [samples](samples). As the framework reaches a more mature state, documentation on the usage of the framework will be added.
+While the framework is still in early development the best way to know what API the framework exposes is by looking at the [tests](src/__tests__), the [source code](src) or the [test](test). As the framework reaches a more mature state, documentation on the usage of the framework will be added.
```

Comment:
```suggestion
While the framework is still in early development the best way to know what API the framework exposes is by looking at the [unit tests](src/__tests__), the [source code](src) or the [e2e tests](test). As the framework reaches a more mature state, documentation on the usage of the framework will be added.
```

Reply:
Good point I agree with `or the [e2e tests](test)` but I wouldn't call all tests in `src/__tests__` as unit tests.

Reply:
Sure. Just having `test` and `tests` is confusing.